src/core/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './LoadingSpinner.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { VisuallyHidden } from '../VisuallyHidden/VisuallyHidden';
+import { ComponentIcon } from '../StaticIcon/StaticIcon';
+import { Icon } from '../../core/Icon/Icon';
+
+export type LoadingSpinnerStatus = 'loading' | 'success' | 'failed';
+export interface LoadingSpinnerProps {
+  /** Custom class name for styling and customizing */
+  className?: string;
+  /** Unique id */
+  id?: string;
+  /** Status text */
+  text: string;
```

Comment:
Should this be named `statusText` since that's what it essentially is? Or does it clash with the `statusText` props used elsewhere for a slightly different purpose? In any case it might make sense to make the prop name and/or description a bit more informative.

Reply:
We had the discussion about this yesterday and the previous label was not suitable as this text is changing based on the status text.
But as you noted the `statusText` as a prop name sounded confusing as it behaves differently than other components.

Reply:
It used to be LabelText.  That was confusing so in a daily meeting we decided to change it to "text". statusText was also in the table, but somehow we ended up with "text".

Reply:
Makes sense. I think text conveys the purpose well enough :)